docs/Quality.md

```diff
@@ -94,13 +94,13 @@ Good details in a bug report include:
   * Trivial (eg. a copy error)
 
 ### Agile Manual QA Overflow
-Developing automated unit or UI tests are time consuming. It's almost impossible to maintain automation tests while a feature is being developed on at the same time. And often having enough time to develop unit tests and UI automation tests while developing is a luxury we don't get. So, to continue to maintain high quality, manual QA becomes necessary until the feature settles down.
+Developing automated unit or UI tests are time consuming. It's almost impossible to maintain automation tests while a feature is being developed. And often, having enough time to create tests while developing is a luxury we don't get. So, to continue to maintain high quality, manual QA becomes necessary until the feature settles down.
```

Comment:
this looks good, although i would like to discuss our testing goals in more detail as a team, specifically in regards to "creat[ing] tests while developing is a luxury we don't get."

Reply:
Think I'm wrong to mention unit tests as a luxury. I'll remove unit tests out of that luxury and replace it with integration tests.